### PR TITLE
Log out functionality spec passing

### PR DESCRIPTION
**Pull request by madhalle**

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -19,4 +19,13 @@ def create
       render :new
     end
   end
+
+  def destroy
+    session.delete(:user_id)
+    session.delete(:cart)
```
Comment (madhalle) [inline app/controllers/sessions_controller.rb:25]: unsure of whether or not we should be saving these session or if they are being saved for when use logs in again

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -14,11 +14,11 @@
 
       <%= link_to "All Merchants", "/merchants" %>
       <%= link_to "All Items", "/items" %>
- 
+
       <%= link_to "Register", "/register" unless current_user %>
       <%= link_to "Log in", "/login" unless current_user %>
       <%= link_to "My Profile", "/profile" if current_user %>
-      <%= link_to "Log Out", "/logout" if current_user %>
+      <%= link_to "Log Out", logout_path, method: "delete" if current_user %>
```
Comment (aperezsantos) [inline app/views/layouts/application.html.erb:21]: Great use of a helper method

File: app/controllers/sessions_controller.rb (hunk 1 above)
Comment (aperezsantos) [reply app/controllers/sessions_controller.rb:25]: Depending on what the user story calls for at this moment, deleting is alright. Perhaps we'll have to revisit this, and change it later.

File: spec/features/login_logout/logout_spec.rb [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    fill_in :email, with:"bigshrek12@castle.co"
+    fill_in :password, with:"boom"
+
+    click_button "Log In"
+
+
+    click_link "Log Out"
+    expect(current_path).to eq("/")
+
+    # visit "/logout"
```
Comment (aperezsantos) [inline spec/features/login_logout/logout_spec.rb:43]: Great testing! Let's clear this up later.

File: spec/features/login_logout/login_spec.rb [hunk 4]
```diff
@@ -78,5 +78,29 @@
       expect(current_path).to eq("/login")
       expect(page).to have_content("Sorry, your credentials are bad.")
     end
+
+    # it "will display an error flash if user is already logged in" do
```
Comment (aperezsantos) [inline spec/features/login_logout/login_spec.rb:82]: Does this need to me commented out?

**Pull request merged**